packages/test/test-utils/src/testFluidObject.ts

```diff
@@ -163,6 +170,15 @@ export class TestFluidObjectFactory implements IFluidDataStoreFactory {
 	constructor(
 		private readonly factoryEntries: ChannelFactoryRegistry,
 		public readonly type = "TestFluidObjectFactory",
+		private readonly dataObjectKind: new (
```

Comment:
This is temporary until `TestFluidObject` is removed right? I find this pattern a little complicated to parse.

Reply:
Possibly. Or we could keep it if there are other use cases for replacing what data object TestFluidObject uses.

I have factored out and documented this type for clarity.